spec/factories/vm_azure.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :vm_azure, :class => "ManageIQ::Providers::Azure::CloudManager::Vm", :parent => :vm_cloud do
     location "westus"
-    vendor   "azure"
+    vendor   "microsoft"
```

Comment:
Opened [Bug 1291922](https://bugzilla.redhat.com/show_bug.cgi?id=1291922) to address setting vendor column properly.  This PR needs to be updated based on the proper vendor value for Azure VMs.